doc/review-checklist.adoc

```diff
@@ -55,12 +76,85 @@ It's not required anymore by the Eclipse Foundation and we may remove this requi
 The code should have a proper copyright which should be updated when necessary.
 
 A commit fixing an issue should also update the `CHANGELOG` to indicate what has been done.
-For most contributions, a simple sentence with the link of the issue and it's description is enough.
+For most contributions, a simple sentence indicating what has been done with the link of the issue is enough.
 Important API breaks should also be noted in the dedicated sections.
 
+IMPORTANT: If a contribution changes code in Sirius Web, most of the time it is to adapt the code to a change in Sirius Components.
+You can pretty much consider any contribution with a change in Sirius Web as having API breaks which should be described separately than the work being done in the CHANGELOG.
+
 Major evolutions require a dedicated `ADR` which should be referenced in the `CHANGELOG` too.
 It is better to contribute the ADR before starting to work on an issue in order to ensure that you are not starting in a wrong direction.
```

Comment:
contribute to the ADR

Reply:
No I meant contributing the ADR relevant to the issue.